WordPress/build.gradle

```diff
@@ -124,6 +127,7 @@ android {
             applicationId "org.wordpress.android.prealpha"
             dimension "buildType"
             buildConfigField "boolean", "MODAL_LAYOUT_PICKER", "true"
+            buildConfigField "boolean", "HOME_PAGE_PICKER", "true"
         }
     }
 
```

Comment:
Note ([ref](https://github.com/wordpress-mobile/WordPress-Android/pull/12482/files#r462066562)):
- `wasabi` is for developers
- `jalapeno` is for reviewers (automatic CI builds on PRs - needs to be manually downloaded and installed)
- `zalpha` is for alpha testers (closed track distributed through Google Play)
- `vanilla` is for beta testers and users/release (open track + release distributed through Google Play)